planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/src/goal_planner_module.cpp

```diff
@@ -875,8 +883,15 @@ BehaviorModuleOutput GoalPlannerModule::plan()
 {
   universe_utils::ScopedTimeTrack st(__func__, *time_keeper_);
 
+  if (!context_data_) {
+    RCLCPP_ERROR(getLogger(), " [pull_over] plan() is called without valid context_data");
+  }
+  const auto & context_data = context_data_.has_value()
+                                ? context_data_.value()
+                                : PullOverContextData(std::make_pair(true, true));
```

Comment:
```suggestion
  const auto context_data = context_data_.has_value()
                                ? context_data_.value()
                                : PullOverContextData(std::make_pair(true, true));
```
temporary object should not be bound